app/app.go

```diff
@@ -607,6 +637,16 @@ func NewMigalooApp(
 	var transferStack porttypes.IBCModule
 	transferStack = transfer.NewIBCModule(app.TransferKeeper)
```

Comment:
@sontrinh16 -- doesn't this do the same thing as what you just added in your PR?

Reply:
hmmm no, i think this func is for setting the actual value of the `TransferKeeper` of the `PFMKeeper` ( `RouterKeeper` in this case ) since the TransferKeeper is intialize after the PFMKeeper so the TransferKeeper we pass in [line 524](https://github.com/White-Whale-Defi-Platform/migaloo-chain/blob/ed23688f5ee208ff4320330fd4761fe984527e65/app/app.go#L524) are empty so we need to call `SetTransferKeeper` to set the newly initialized value of `TranferKeeper`